x/inflation/spec/01_concepts.md

```diff
@@ -52,20 +52,17 @@ The inflation distribution for staking, usage incentives and community pool is
 implemented through an exponential formula, a.k.a. the Half Life.
 
 Inflation is minted in daily epochs. During a period of 365 epochs (one year), a
-daily provision of Evmos tokens is minted and allocated to staking rewards,
-usage incentives and the community pool (`epochProvison`). The epoch provision
-doesn’t change within a period and its cumulated amount per period is equal
-total inflation per period (`epochProvision * epochsPerPeriod =
-periodProvision`).
+daily provision (`epochProvison`) of Evmos tokens is minted and allocated to staking rewards,
+usage incentives and the community pool. The epoch provision depends on parameters and is calculated every day.
```

Comment:
I think it makes sense to mention here that the parameters are controlled by governance.
```suggestion
usage incentives and the community pool. The epoch provision depends on governance-controlled parameters and is calculated every day.
```

Reply:
I think this it's obvious as parameters of every module controlled by governance, I would rather add word `module`, to get `on module parameters`

Reply:
module parameters sounds good to me as well